api/fasit.go

```diff
@@ -61,6 +63,53 @@ func (fasit FasitClient) GetResources(resourcesRequests []ResourceRequest, envir
 	return resources, nil
 }
 
+func (fasit FasitClient) createOrUpdateFasitResources(exposedResourceIdsRef *[]int, resources []ExposedResource, hostname, environment, application, zone string) (error) {
+	exposedResourceIds := *exposedResourceIdsRef
```

Comment:
I dont understand the reason for this line

Reply:
maybe easier to understand if the code returns a slice of resourceids

Reply:
for this to work you have to return the result from append
+1 for simplicity, and just return a fresh slice of ints